WinFormsUI/TransactionEditor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Core;
+using System;
+using System.Windows.Forms;
+
+namespace WinFormsUI
+{
+    public partial class TransactionEditor : Form
```

Comment:
Codacy found an issue: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6066087)